mezzanine/core/static/mezzanine/js/jquery-ui-1.12.1.js

```diff
@@ -1,18706 +0,0 @@
-/*! jQuery UI - v1.12.1 - 2016-09-14
```

Comment:
jQuery is still being used in the admin and on the inline editor. As you add or remove files make sure you test the admin (specifically the admin page tree and admin inlines). You will also need to test the frontend editor by login in as admin and using the little "Edit" tags (see screenshot). This feature is documented here: http://mezzanine.jupo.org/docs/inline-editing.html

We definitely want to get rid of jQuery for regular users, but admin users will still need to load it for the inline-editor.

![image](https://user-images.githubusercontent.com/2391102/90318551-7da88400-deee-11ea-8bd0-8f5180a17a1f.png)